packages/k6/src/repro-test.ts

```diff
@@ -0,0 +1,71 @@
+import { check, sleep } from 'k6'
+import { SharedArray } from 'k6/data'
+import http, { BatchRequests } from 'k6/http'
+import { Rate } from 'k6/metrics'
+import { Payload } from './config/types'
+
+// load the test iterations from the environment or default to 100
+let iterations = 100
+if (__ENV.LOAD_TEST_ITERATIONS) {
+  iterations = parseInt(__ENV.LOAD_TEST_ITERATIONS)
+}
+
+// set the k6 running options
+export const options = {
+  vus: 1,
+  iterations,
+  thresholds: {
+    http_req_failed: ['rate<0.01'], // http errors should be less than 1%
+    http_req_duration: ['p(95)<200'], // 95% of requests should be below 200ms
+  },
+}
+
+export const errorRate = new Rate('errors')
+
+// load the test data, if data was generated then load it from the generated file
+if (!__ENV.PAYLOAD_PATH) {
+  console.error('PAYLOAD_PATH env var is not present')
+  process.exit(1)
+}
```

Comment:
Is it not safe to default `PAYLOAD_PATH` to `'../src/config/http.json'`?

Reply:
For now this is separate from the other testing workflow, in the future this file could also serve to perform the same tests as the other